src/lib/common/Kconfig

```diff
@@ -337,11 +336,26 @@ choice
           disabled in compile time and won't be present in the
           final library.
 
+          It only affects log levels in the library functions.
+          If an application is using Soletta log system, it
+          needs to be changed using application CFLAGS.
+
```

Comment:
maybe say the `CFLAGS += -DSOL_LOG_LEVEL_MAXIMUM=2` to disable all log levels greater than 2 (warning)?

I'd also add that same remark to sol-log.h inside the `SOL_LOG_LEVEL_MAXIMUM` documentation.